src/main/java/uk/gov/justice/digital/hmpps/whereabouts/controllers/CourtController.kt

```diff
@@ -56,4 +60,11 @@ class CourtController(private val courtService: CourtService) {
   @ApiOperation(value = "Create a Video Link Booking")
   fun createVideoLinkBooking(@RequestBody @Valid videoLinkBookingSpecification: VideoLinkBookingSpecification) =
     courtService.createVideoLinkBooking(videoLinkBookingSpecification)
+
+  @PostMapping(path = ["/appointment-linker"])
+  @ResponseStatus(HttpStatus.NO_CONTENT)
+  @ApiOperation(value = "Create Video Link Bookings for dangling Video Link Appointments")
+  fun linkDanglingAppointments() {
```

Comment:
Should this be guarded by membership of some role?

Reply:
Feels like it should but then again I suspect this code will eventually be removed once all the appointments have been linked?